src/app/modules/item/components/item-permissions/item-permissions.component.html

```diff
@@ -81,16 +81,26 @@
           Note that some subgroups or users of the groups may have higher permissions.
```

Comment:
I think tooltip text should be either centered or justified.
@smadbe ?
<img width="250" alt="screenshot" src="https://user-images.githubusercontent.com/16257159/175886830-bf2ba6ce-8e7e-43a2-8258-2ec76e24f38b.png">

Reply:
Or just using a larger width so that it does not appear as a long text.

Reply:
Do whatever is possible with primeng without big changes

Reply:
Fixed